src/components/ModalAccessByHardware.vue

```diff
@@ -159,7 +162,7 @@ export default createComponent({
         );
 
         const checkIsChrome = computed(() => {
-            return (state.isChrome = navigator.userAgent.includes("Chrome"));
+            return (state.isChrome = browser === "Chrome");
```

Comment:
This should be just a constant. Not even a reactive property. That gets returned in the context object.